payloads/v10/poll.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/resources/poll
+ */
+
+import type { APIPartialEmoji } from './emoji';
+
+/**
+ * https://discord.com/developers/docs/resources/poll#poll-object-poll-object-structure
+ */
+export interface APIPoll {
+	/**
+	 * The question of the poll
+	 */
+	question: APIPollMedia;
+	/**
+	 * Each of the answers available in the poll, up to 10
+	 */
+	answers: APIPollAnswer[];
+	/**
+	 * The time when the poll ends (IS08601 timestamp)
+	 */
+	expiry: string;
+	/**
+	 * Whether a user can select multiple answers
+	 */
+	allow_multiselect: boolean;
+	/**
+	 * The layout type of the poll
+	 */
+	layout_type: PollLayoutType;
+	/**
+	 * The results of the poll
+	 */
+	results: APIPollResults;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/poll#layout-type
+ */
+export enum PollLayoutType {
+	/**
+	 * The, uhm, default layout type
+	 */
+	Default = 1,
+}
+
+/**
+ * https://discord.com/developers/docs/resources/poll#poll-media-object-poll-media-object-structure
+ */
+export interface APIPollMedia {
+	/**
+	 * The text of the field
+	 *
+	 * The maximum length is `300` for the question, and `55` for any answer
+	 */
+	text?: string;
+	/**
+	 * The emoji of the field
+	 */
+	emoji?: APIPartialEmoji;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/poll#poll-answer-object-poll-answer-object-structure
+ */
+export interface APIPollAnswer {
+	/**
+	 * The ID of the answer. Starts at `1` for the first answer and goes up sequentially
+	 *
+	 * Only sent as part of responses from Discord's API/Gateway
```

Comment:
Let's get rid of this and change the `RESTAPIPollCreate` interface to not have this prop